core/object.h

```diff
@@ -111,6 +111,7 @@ enum PropertyUsageFlags {
 	PROPERTY_USAGE_CLASS_IS_ENUM = 1 << 18,
 	PROPERTY_USAGE_NIL_IS_VARIANT = 1 << 19,
 	PROPERTY_USAGE_INTERNAL = 1 << 20,
+	PROPERTY_USAGE_NO_DUPLICATE = 1 << 21, // If duplicated this property is not copied
```

Comment:
I would rename to something like
PROPERTY_USAGE_DO_NOT_SHARE_ON_DUPLICATE

would also go well in AnimationPlayer for the animations